classes/ColdTrick/EventManager/Bootstrap.php

```diff
@@ -6,7 +6,7 @@
 use Event;
 
 class Bootstrap extends DefaultPluginBootstrap {
-	
```

Comment:
could you try to configure your editor to not trim blank lines? otherwise it will polute PR's